src/main/api/allApi.js

```diff
@@ -0,0 +1,7 @@
+const API_BASE_URL_FOLDER_ALL = 'https://bootcamp-api.codeit.kr/api/users/1/links';
```

Comment:
```suggestion
const API_BASE_URL = 'https://bootcamp-api.codeit.kr' //외부에 선언하고  import하는 형식
const API_FOLDER_ALL = `${API_BASE_URL }/api/users/1/links`;
```

[중요도: 중간] API_BASE_URL을 포함한 정보를 한번에 선언하는 거라면 의미가 크게 있어 보이지는 않습니다.
[중요도: 높음] 혹시 /api/users/1/links에 있는 숫자 1은 직접 주입하신 걸까요? 아니면 본인이 넣은걸까요? 만약 본인이 넣은거라면 수정이 필요해보입니다. 서버에서 가져온 정보면 아마 localStorage 혹은 저장된 유저 정보를 import해서 쓰는게 좋아보입니다. (어찌보면 상태관리의 개념으로 넘어갈수도...)

밑에도 내용은 비슷해 밑 파일의 내용들은 생략하겠습니다.

Reply:
수정 후 파일을 하나로 통합했습니다!